go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/labstack/gommon v0.4.0 // indirect
 	github.com/mattn/go-colorable v0.1.13 // indirect
 	github.com/mattn/go-isatty v0.0.19 // indirect
+	github.com/otiai10/copy v1.12.0
```

Comment:
`go mod tidy` is needed here - indirect and direct dependencies are automatically split into separate groups as of a few Go versions ago, so this is in the wrong place